### ISSUE TITLE
Screen reader not reading text with angle bracket in it

### ISSUE THREAD

**Issue by ashishguptaiitb** [MicrosoftDocs/microsoft-style-guide#128]
Please check if we need a different approach than bold or angle brackets from an a11y perspective. When I try reading any text with bold using my Windows 10 Narrator, it doesn't read the text. In fact, it stops before the bold text and doesn't let me select it with previous text. I can select bold words individually. 
An example, in case of last paragraph the Narrator starts after the bold text and speaks, "Menu go to folder, double tap to activate". It doesn't read after the comma as well, even though I've made sure it is selected. But this is a different matter. 

My question is: if bold text cannot be read as easily with in-built Narrator then should we writers bold text from an accessibility perspective?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 39b5e3aa-271d-e55e-5c7c-fd31ec3ae360
* Version Independent ID: 18c8323f-fa16-669e-2d58-f959b8b772d7
* Content: [Writing step-by-step instructions - Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/writing-step-by-step-instructions#feedback)
* Content Source: [styleguide/procedures-instructions/writing-step-by-step-instructions.md](https://github.com/MicrosoftDocs/microsoft-style-guide/blob/master/styleguide/procedures-instructions/writing-step-by-step-instructions.md)
* Product: **non-product-specific**
* GitHub Login: @pallep
* Microsoft Alias: **pallep**
